.github/buildomat/jobs/deploy.sh

```diff
@@ -201,11 +201,16 @@ routeadm -e ipv4-forwarding -u
 PXA_START="$EXTRA_IP_START"
 PXA_END="$EXTRA_IP_END"
 
-# These variables are used by softnpu_init, so export them.
-export GATEWAY_IP GATEWAY_MAC PXA_START PXA_END
-
 pfexec zpool create -f scratch c1t1d0 c2t1d0
-VDEV_DIR=/scratch ptime -m pfexec ./tools/create_virtual_hardware.sh
+
+ptime -m \
+    cargo xtask virtual-hardware \
```

Comment:
There's no cargo in the lab environment, right?  Presumably you need to build the xtask binary and include that in whatever job output become inputs for the deploy job

Reply:
Good catch, updated to take this approach